docs/integration-guides/advanced.md

```diff
@@ -290,6 +290,54 @@ To subscribe to all confirmed blocks:
 }
 ```
 
+Details about the election leading to the confirmation can be obtained by setting the `include_election_info` option to true:
+
+```json
+{
+  "action": "subscribe",
+  "topic": "confirmation",
+  "options": {
+    "include_election_info": "true"
+  }
+}
+```
+
+Below is a full message which includes the following election info:
+
+* election duration in milliseconds
+* election timestamp as milliseconds since epoch
```

Comment:
Can we specify if it's the start or end of the election?

Reply:
Yep, added